sphinx/ext/napoleon/docstring.py

```diff
@@ -1131,6 +1131,11 @@ def _parse_numpydoc_see_also_section(self, content: List[str]) -> List[str]:
         func_name1, func_name2, :meth:`func_name`, func_name3
 
         """
+        try:
+            inventory = self._app.builder.env.intersphinx_inventory
+        except AttributeError:
+            inventory = {}
```

Comment:
is there any way to silence `mypy`? I'm aware that `BuildEnvironment` may not have a attribute named `intersphinx_inventory`, that's why I'm catching `AttributeError` (it's also easier to make the tests work that way)...

Reply:
I simply added a `# type: ignore` comment on that line, but I'm not sure if that was the best way to fix this?